tests/integration/test_examples.py

```diff
@@ -13,6 +13,7 @@
 #  permissions and limitations under the License.
 import shutil
 from pathlib import Path
+from typing import Dict
```

Comment:
Didn't we decide to go away from this `Dict` stuff to `dict`? I remember @schustmi saying that we should just use `dict` but there seems to have been a change for some reason. Maybe I missed it though

Reply:
See my answer above, here with the official documentation (https://docs.python.org/3.9/library/typing.html#typing.Dict)